### PR TITLE
resolving some nitpicks from pr #1245

### PR DESCRIPTION
**Pull request by lbergelson**
resolving some nitpicks left over from pr #1245

### CODE REVIEW

**Comment by romanzenka** [comment]
Looks like there is a tiiiny coverage drop thanks to an exception being thrown when stream does not have "markSupported". I can try to make a test case for this if you want me to.

File: src/main/java/htsjdk/samtools/util/IOUtil.java [hunk 1]
```diff
@@ -99,6 +99,8 @@
     public static final String COMPRESSED_VCF_INDEX_EXTENSION = ".tbi";
 
 
+
```
Comment (yfarjoun) [inline src/main/java/htsjdk/samtools/util/IOUtil.java]: why extra nl?

File: src/main/java/htsjdk/variant/utils/VCFHeaderReader.java [hunk 2]
```diff
@@ -50,7 +51,7 @@ public static VCFHeader readHeaderFrom(final SeekableStream in) throws IOExcepti
     private static InputStream bufferAndDecompressIfNecessary(final InputStream in) throws IOException {
         BufferedInputStream bis = new BufferedInputStream(in);
         // despite the name, SamStreams.isGzippedSAMFile looks for any gzipped stream (including block compressed)
```
Comment (yfarjoun) [inline src/main/java/htsjdk/variant/utils/VCFHeaderReader.java]: update comment

File: src/main/java/htsjdk/samtools/util/IOUtil.java (hunk 1 above)
Comment (lbergelson) [reply src/main/java/htsjdk/samtools/util/IOUtil.java]: accident...

File: src/main/java/htsjdk/variant/utils/VCFHeaderReader.java (hunk 2 above)
Comment (lbergelson) [reply src/main/java/htsjdk/variant/utils/VCFHeaderReader.java]: done

**Pull request merged**